pkg/controllers/disruption/emptynodeconsolidation.go

```diff
@@ -96,6 +96,10 @@ func (c *EmptyNodeConsolidation) ComputeCommand(ctx context.Context, disruptionB
 	// Get the current representation of the proposed candidates from before the validation timeout
 	// We do this so that we can re-validate that the candidates that were computed before we made the decision are the same
 	candidatesToDelete := mapCandidates(cmd.candidates, validationCandidates)
+	if invalidatedCount := len(cmd.candidates) - len(candidatesToDelete); invalidatedCount != 0 {
+		logging.FromContext(ctx).Debugf("abandoning empty node consolidation attempt, %d candidates are no longer valid, command is no longer valid, %s", invalidatedCount, cmd)
```

Comment:
Can we add these formatted directives into structured values in the logger?
```suggestion
		logging.FromContext(ctx).Debugf("abandoning empty node consolidation attempt, %d candidates are no longer valid, command is no longer valid, %s", invalidatedCount, cmd)
```